src/dmd/statementsem.d

```diff
@@ -4585,45 +4585,69 @@ Statement scopeCode(Statement statement, Scope* sc, out Statement sentry, out St
     return statement;
 }
 
-
 /*******************
  * Determines additional argument types for makeTupleForeach.
  */
 static template TupleForeachArgs(bool isStatic, bool isDecl)
```

Comment:
Isn't this unused now?

Reply:
No, it's still used.

Reply:
So we should probably update the documentation? Otherwise, people will look into `makeTupleForeach` and probably get confused.

Reply:
The whole maketuple stuff is confusing. I plan on straightening it out.